eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -183,44 +181,34 @@ spoAccepted re rs gas =
 --  * `a` - total delegated stake that voted Abstain
 --  * `s` - total delegated stake
 --
--- However, computing the total stake again would be wasteful, since we already have
--- distributions per pool computed. So, values that we have available are not exactly
--- what we need, because we have:
---  * `y/s` - stake that votes yes over the total stake
---  * `a/s` - stake that voted abstain over the total stake
---
--- We divide both numerator and denominator by `s` and we'll get a formula that we can
--- use:
---
--- t = y \/ (s - a) = (y \/ s) / (1 - a \/ s)
---
 -- For `HardForkInitiation` all SPOs that didn't vote are considered as
 -- `No` votes. Whereas, for all other `GovAction`s, SPOs that didn't
 -- vote are considered as `Abstain` votes.
+--
+-- `No` votes are not counted.
 spoAcceptedRatio :: forall era. RatifyEnv era -> GovActionState era -> Rational
 spoAcceptedRatio
-  RatifyEnv {reStakePoolDistr}
+  RatifyEnv {reStakePoolDistr = PoolDistr individualPoolStake (CompactCoin totalActiveStake)}
   GovActionState
     { gasStakePoolVotes
-    , gasProposalProcedure =
-      ProposalProcedure {pProcGovAction}
+    , gasProposalProcedure = ProposalProcedure {pProcGovAction}
     }
-    | abstainVotesRatio == 1 = 0 -- guard against the degenerate case when all abstain.
-    | otherwise = yesVotesRatio / (1 - abstainVotesRatio)
+    | totalActiveStake == 0 = 0 -- guard against the degenerate case when active stake is zero.
```

Comment:
I'm not sure this is necessary, because the next case pretty much implies this anyways, but it doesn't hurt either to have it here.

Reply:
Yes, I have just added both :+1: